lib/shared/addon/components/new-select/component.js

```diff
@@ -74,9 +75,11 @@ export default Component.extend({
   },
 
   // dont know why this is here but it causes issues trying to set on undefined objects
-  // didRender() {
-  //   this._change();
-  // },
+  didRender() {
+    if (this.get('setValueOnInit')) {
```

Comment:
What issue was this change for?  This seems like it's going to break anywhere that there's a value that's no longer in the list of choices.